insight-engine/latest/config/indexing.md

```diff
@@ -0,0 +1,56 @@
+---
+title: Indexing recommendations
+---
+
+When upgrading from a previous Insight Engine version you should review these indexing considerations:
+
+* [Cross Locale](cross-locale) Enabling or disabling
+* [Exact term search](exact-term-search)
+* [Document fingerprints]({% link insight-engine/latest/config/performance.md %}#disable-document-fingerprints) Enabling or disabling
+
+## Cross Locale
+
+By default, Insight Engine is provided with cross-language search support disabled. This default configuration affects all the deployment artifacts, i.e. .zip file and Docker images.
```

Comment:
I would say "zip file" rather than ".zip file"

Reply:
Fixed. for both SS and SIE